pkg/filter/expr_filter.go

```diff
@@ -405,20 +401,21 @@ func (f *dmlExprFilter) getRules(schema, table string) []*dmlExprFilterRule {
 
 // shouldSkipDML skips dml event by sql expression.
 func (f *dmlExprFilter) shouldSkipDML(
-	row *model.RowChangedEvent,
-	rawRow model.RowChangedDatums,
-	ti *model.TableInfo,
+	dmlType common.RowType,
+	preRow, row chunk.Row,
+	ti *common.TableInfo,
```

Comment:
Please use `tableInfo` instead of `ti` in all related codes.

Reply:
good suggestion, done